website/project/model.py

```diff
@@ -3815,6 +3841,8 @@ class DraftRegistration(StoredObject):
 
     approval = fields.ForeignField('draftregistrationapproval', default=None)
 
+    status_logs = fields.ForeignField('draftregistrationlog', list=True)
```

Comment:
>Does having a list of logs imply a new set of automatic queries? 

Assuming I understand your question, you could still have a property on this model that just returns `DraftRegistrationLog.find(Q('owner', 'eq', self._id))` or similar. You probably shouldn't need to change `'logs': map(serialize_draft_logs, draft.status_logs)` above.

> If so, why do we even have this option? In what case would this ever be optimal?

In the case where there exist a large number of DRLogs, you gain the benefit of not needing to load each into memory when loading the DraftRegistration.

> Why is a backref, assuming the previous question is reasonable, any better? 

It's not a *backref*, exactly. It's a forward ref in the other direction.

> Won't it load the DraftRegistration with each log loaded?

Yes, which is better than loading every DRLog when the associated DraftRegistration is loaded -- one reason is above. Another reason is that the DR will likely be cached already if you're loading a DRLog; the reverse does not necessarily hold true.